cv/francisco.schallibaum.md

```diff
@@ -0,0 +1,30 @@
+# Curriculum Vitae
+
+## About me
+- JUAN FRANCISCO SCHALLIBAUM
+I'm a Psychology graduate, and I have worked on this field for 5+
+years, until I decided to make a career change. I'm actually learning
+the technology up-to-date standards in the Frontend Development
+area. I'm looking for Jr. or trainee opportunities so as to learn in a
+collaborative environment.
+
+## Contact me
+- Address: 73 street N°1222 CP1900 (La Plata)
+- Phone: +54 011 23864313
+- Mail: jfschallibaum@gmail.com
+- Github: https://github.com/SonicFran99
+- LinkedIn: https://www.linkedin.com/in/juan-francisco-schallibaum-886530225/
+
+## Skills
+- JavaScript (intermidate/advanced)
+- HTML5 (advanced)
+- CSS3 (intermidiate/advanced)
+- Bootstrap (intermidiate)
```

Comment:
There's a minor typo here, it should be 'intermediate', but the rest looks good!